std-bits/base/src/main/java/org/enso/base/cache/DiskSpaceGetter.java

```diff
@@ -4,8 +4,8 @@
 import org.enso.base.CurrentEnsoProject;
 
 public class DiskSpaceGetter extends Mockable<Long> {
-  public DiskSpaceGetter() {
-    super(() -> getRootPath().getUsableSpace());
+  public Long computeValue() {
```

Comment:
I'd probably add `@Override` at the top.

Reply:
Done